docs/lakehouse-generic-lance-table.md

```diff
@@ -0,0 +1,327 @@
+| `lance.creation-mode` | Create mode: for create table, it can be `CREATE`, `EXIST_OK` or `OVERWRITE`. and it should be `CREATE` or `OVERWRITE` for registering tables                                                                                                                                                                                                   | `CREATE` | No           | 1.1.0         |
+| `lance.register`      | Whether it is a register table operation. This API will not create data directory actually and it's the user's responsibility to create and manage the data directory.                                                                                                                                                                          | false    | No           | 1.1.0         |
+| `lance.storage.xxxx`  | Any additional storage-specific properties required by Lance format (e.g., S3 credentials, HDFS configs). Replace `xxxx` with actual property names. For example, we can use `lance.storage.aws_access_key_id` to set S3 aws_access_key_id when using a S3 location, for detail, please refer to https://lancedb.com/docs/storage/integrations/ | (none)   | No           | 1.1.0         |
+
+
+**Location Requirement:** Must be specified at catalog, schema, or table level. See [Location Resolution](./lakehouse-generic-catalog.md#key-property-location).
+
+You may also set additional properties specific to your lakehouse format or custom requirements.
+
+### Index Support
+
+Index capabilities vary by lakehouse format. The following table shows Lance format support:
+
+| Index Type | Description                                                                                  | Lance Support |
+|------------|----------------------------------------------------------------------------------------------|---------------|
+| SCALAR     | Optimizes searches on scalar data types (integers, floats, etc.)                             | ✅            |
+| VECTOR     | Optimizes similarity searches in high-dimensional vector spaces                              | ✅            |
+| BTREE      | Balanced tree for sorted data with logarithmic search/insert/delete complexity               | ✅            |
+| INVERTED   | Full-text search optimization through term-to-location mapping                               | ✅            |
+| IVF_FLAT   | Vector search with inverted file and flat quantization                                       | ✅            |
+| IVF_SQ     | Vector search with scalar quantization for memory efficiency                                 | ✅            |
+| IVF_PQ     | Vector search with product quantization balancing accuracy and memory                        | ✅            |
+
+:::caution Index Creation Limitation
+**Lance tables do not support index creation during table creation.** You must:
+1. Create the table first
+2. Then create indexes on the created table
```

Comment:
How do users do this?

Reply:
You should tell users how to do this. AFAIK, is there any API for the user to create an index?

Reply:
Considering creating a lance index is not GA(generic catalog does support ALTER TABLE to add an index, but it's not stable), I will remove the `index` chapter.